libs/backend/src/types/games.ts

```diff
@@ -0,0 +1,4 @@
+import { GameEventUserInfo } from "types";
+
+export type WebSocketGame = Map<string, GameEventUserInfo>;
```

Comment:
GameEventUserInfo should be renamed to just GameUserInfo, the event is in between there, but has no business there, yes it's send by GameEvents, but feels fairly illogical and crossing borders into territory that could be seen as predatory UwU